Source/Base/ASAvailability.h

```diff
@@ -50,8 +50,13 @@
   #define __IPHONE_11_0 110000
 #endif
 
+#ifndef __IPHONE_13_0
+  #define __IPHONE_13_0 130000
+#endif
+
 #define AS_AT_LEAST_IOS10  (kCFCoreFoundationVersionNumber >= kCFCoreFoundationVersionNumber_iOS_10_0)
 #define AS_AT_LEAST_IOS11  (kCFCoreFoundationVersionNumber >= kCFCoreFoundationVersionNumber_iOS_11_0)
+#define AS_AT_LEAST_IOS13  (__IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_13_0)
```

Comment:
Will this have to change to look like the previous ones at some point?

Reply:
we don't think so since it seems like apple hasn't updated kCFCoreFoundationVersionNumber in a long time